thoth/package_extract/image.py

```diff
@@ -342,14 +342,17 @@ def _ld_config_entries(path: str) -> Generator[str, None, None]:
                     if not line:
                         continue
 
-                    entry_path = os.path.join(path, line)
-                    if os.path.isdir(entry_path):
-                        yield line
-                    elif os.path.isfile(entry_path):
-                        # ld.so.conf can point to another configuration files.
-                        stack.append(line)
-                    else:
-                        _LOGGER.warning("Skipping entry %r from symbols extraction not a file or directory", line)
+                    for entry_path in glob.glob(os.path.join(path, relative_path, line)):
```

Comment:
Last thing:
```python
if line.startswith("/"):
    to_glob = os.path.join(path, line[1:])
else:
    to_glob = os.path.join(path, relative_path, line)
```

Reply:
Forgot about this one. Nice catch! Should be fixed in 09c5671